src/features/unified-share-modal/UnifiedShareForm.js

```diff
@@ -117,10 +117,12 @@ class UnifiedShareForm extends React.Component<USFProps, State> {
             .then(({ classificationLabelId, options }: GetJustificationReasonsResponse) => {
```

Comment:
I think we should update `GetJustificationReasonsResponse` that `options` is nullable

Reply:
Could probably also do `options = {}` to simplify things a bit.

Reply:
done both, but went with @jpfranco 's suggestion `options = []`. Using `{}` resulted in `options.map is not a function` error

Reply:
Whoops, my bad. I think my brain automatically parsed "options" as an object. Sorry about that.